analyzer/tests/libtest/codechecker.py

```diff
@@ -142,6 +142,20 @@ def check_force_ctu_capable(is_capable):
     return is_capable
 
 
+def check_force_extdef_mapping_can_read_pch():
+    """
+    Returns True if extdef mapping is force enabled by the
+    'CC_TEST_FORCE_EXTDEF_MAPPING_CAN_READ_PCH' environment variable.
+    """
+    try:
+        return bool(util.strtobool(
+            os.environ['CC_TEST_FORCE_EXTDEF_MAPPING_CAN_READ_PCH']))
```

Comment:
@bruntib TODO set a JIRA ticket to turn this variable on before our next internal release, when we rebase Clang.

Reply:
This should be done before the PR is merged.